packages/base-driver/lib/express/ssl-handler.js

```diff
@@ -0,0 +1,66 @@
+import {readFileSync} from 'fs';
+import {logger} from '@appium/support';
+
+const log = logger.getLogger('Appium');
+
+class SslHandler {
```

Comment:
it sort of feels like this could all simply be a single async function rather than a class. after we construct the httpsoptions we're not doing very much with this class. it doesn't have behaviours on it really.